subprojects/diagnostics/src/main/java/org/gradle/api/tasks/diagnostics/DependencyInsightReportTask.java

```diff
@@ -393,63 +452,144 @@ private AttributeContainer concat(AttributeContainer configAttributes, Attribute
                     ((AttributeContainerInternal) dependencyAttributes).asImmutable());
         }
 
-        private void writeAttributeBlock(StyledTextOutput out, AttributeContainer attributes, AttributeContainer requested) {
-            out.withStyle(Description).text(" [");
+        private void writeAttributeBlock(StyledTextOutput out, AttributeContainer attributes, AttributeContainer requested, boolean selected) {
+            out.withStyle(Description).text("  Attributes:");
```

Comment:
Some tweaks to the output:
- Let's color the variant name and not the word Variant (green = selected, yellow = not selected, red = incompatible?)
- Let's drop the quotes around the variant name
- Let's drop the "Attributes" header and change the table to say "Attribute Name".  That eliminates a few lines and let's us reduce indentation a bit.